From bf24da87569cf17df49cda4a828226a078f5d015 Mon Sep 17 00:00:00 2001
From: sashko9807 <sashko506@gmail.com>
Date: Sat, 1 Jun 2019 19:19:13 +0300
Subject: [PATCH 2/3] libbt: SCO configuration over PCM interface

Audio configuration for PCM, with SCO_CODEC_CVSD codec used by default

Change-Id: Iff3981c6b0ac18408b456ce9fa481256da2cab14
---
 src/hardware.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/hardware.c b/src/hardware.c
index 81e0721..39f5860 100644
--- a/src/hardware.c
+++ b/src/hardware.c
@@ -1482,6 +1482,8 @@ void hw_sco_config(void)
 
     if (SCO_INTERFACE_I2S == sco_bus_interface) {
         hw_sco_i2spcm_config(SCO_CODEC_CVSD);
+    }else if(SCO_INTERFACE_PCM == sco_bus_interface) {
+        hw_sco_i2spcm_config(SCO_CODEC_CVSD);
     }
 
     if (bt_vendor_cbacks)
-- 
2.17.1

